views/includes/page-templates/add-trust.html

```diff
@@ -1,3 +1,11 @@
+{% macro trustStatus(trust) %}
+  {% if trust.ceased_date_day and trust.ceased_date_month and trust.ceased_date_year %}
+    Removed
+  {% else %}
+    Active
+  {% endif %}
+{% endmacro %}
+
```

Comment:
the macro is essentially a nunjucks function that can be called from within the template